managed/CounterStrikeSharp.API/Modules/Menu/CenterHtmlMenu.cs

```diff
@@ -13,114 +13,73 @@
  *  You should have received a copy of the GNU General Public License
  *  along with CounterStrikeSharp.  If not, see <https://www.gnu.org/licenses/>. *
  */
-
 using System.Text;
-using Microsoft.Extensions.Logging;
 
 namespace CounterStrikeSharp.API.Modules.Menu
 {
     public class CenterHtmlMenu : BaseMenu
     {
-        public CenterHtmlMenu(string title) : base(ModifyTitle(title))
-        {
-        }
-        
-        public override ChatMenuOption AddMenuOption(string display, Action<CCSPlayerController, ChatMenuOption> onSelect, bool disabled = false)
-        {
-            var option = new ChatMenuOption(ModifyOptionDisplay(display), disabled, onSelect);
-            MenuOptions.Add(option);
-            return option;
-        }
-        
-        private static string ModifyTitle(string title)
+        public CenterHtmlMenu(string title) : base(title)
         {
-            if (title.Length > 32)
-            {
-                Application.Instance.Logger.LogWarning("Title should not be longer than 32 characters for a CenterHtmlMenu");
-                return title[..32];
-            }
-
-            return title;
-        }
-
-        private static string ModifyOptionDisplay(string display)
-        {
-            if (display.Length > 26)
-            {
-                Application.Instance.Logger.LogWarning("Display should not be longer than 26 characters for a CenterHtmlMenu item");
-                return display[..26];
-            }
-
-            return display;
         }
     }
 
     public class CenterHtmlMenuInstance : BaseMenuInstance
     {
+        public override int NumPerPage { get; } = 4;
```

Comment:
Why change this to 4 instead of 5?
If it's just so the `quit` option is not half truncated by the bottom of the popup, I think it's best to leave more options.